pkg/sdk/integration_tests/accounts_integration_test.go

```diff
@@ -1,13 +1,13 @@
-package sdk
+package sdk_integration_tests
 
```

Comment:
This package name goes againt golang convention. https://go.dev/blog/package-names. recommend to rename to simply "integration"

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns, such as:
> time (provides functionality for measuring and displaying time)
> list (implements a doubly linked list)
> http (provides HTTP client and server implementations)
> The style of names typical of another language might not be idiomatic in a Go program. Here are two examples of names that might be good style in other languages but do not fit well in Go:
> 
> computeServiceClient
> priority_queue

Reply:
Good point. I do not like simple integration though because I would expect that integration contains some plugins to integrate with some external systems or something like that. We usually start the name of the integration test with `TestInt_` prefix, so maybe let's rename it to `testint`?

Reply:
Done.